examples/graphql-dataloader/graphql_dataloader.md

```diff
@@ -0,0 +1,22 @@
+# GraphQL service - Dataloader
+
+The Ballerina GraphQL module provides the capability to batch and cache data fetching from data sources using the `garaphql.dataloader` submodule. To leverage this functionality in a GraphQL service, users must register dataloaders through the `graphql:Context` object and implement the corresponding prefetch method logic and resolver logic. By using `garaphql.dataloader`, users can efficiently avoid frequent data fetching, effectively addressing the GraphQL N+1 problem.
```

Comment:
We have to add a brief description for the prefetch method here as well.

Reply:
We have added an explanation in the code comment. Do we need to repeat it here ?

Reply:
The description itself should be self-sufficient.

Reply:
Updated with [c6fac31](https://github.com/ballerina-platform/ballerina-distribution/pull/4776/commits/c6fac31fa7a1cad15bf701033ca7e9ba732d5e7d)